rs-react-app/src/services/LocalStorageService.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * LocalStorageService - Handles localStorage operations for the application
+ */
+const SEARCH_TERM_KEY = 'season-search-term';
+
+/**
+ * Save search term to localStorage
```

Comment:
Why have you decided to use JSDoc? How did you choose files which must be covered with annotations and which do not?